src/pages/content/Player.tsx

```diff
@@ -301,10 +301,18 @@ function Player({ room, player }: Props) {
     }
   };
 
-  const handlePlayerVisiblityChange = () => {
-    let curr = localStorage.getItem("42fm:hidePlayer");
-
-    localStorage.setItem("42fm:hidePlayer", curr === "true" ? "false" : "true");
+  const handlePlayerVisibilityChange = () => {
+    const oldValue = localStorage.getItem('42fm:hidePlayer');
```

Comment:
In my opinion, it would be worth considering whether to extract the localStorage keys to some file